packages/mobile/src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -0,0 +1,118 @@
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { Spacing } from '@celo/react-components/styles/styles'
+import { StackScreenProps } from '@react-navigation/stack'
+import React, { useLayoutEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text, View } from 'react-native'
+import * as Keychain from 'react-native-keychain'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch } from 'react-redux'
+import { choseToRestoreAccountSelector } from 'src/account/selectors'
+import { setUseBiometry } from 'src/app/actions'
+import { registrationStepsSelector, supportedBiometryTypeSelector } from 'src/app/selectors'
+import {
+  default as Face,
+  default as FaceID,
+  default as Fingerprint,
+  default as TouchID,
+} from 'src/icons/biometrics/FaceID'
+import { HeaderTitleWithSubtitle, nuxNavigationOptions } from 'src/navigator/Headers'
+import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarTextButton } from 'src/navigator/TopBarButton'
+import { StackParamList } from 'src/navigator/types'
+import { default as useSelector } from 'src/redux/useSelector'
+
+type Props = StackScreenProps<StackParamList, Screens.EnableBiometry>
+
+const biometryImageMap: { [key in Keychain.BIOMETRY_TYPE]: JSX.Element } = {
+  [Keychain.BIOMETRY_TYPE.FACE_ID]: <FaceID />,
+  [Keychain.BIOMETRY_TYPE.TOUCH_ID]: <TouchID />,
+  [Keychain.BIOMETRY_TYPE.FINGERPRINT]: <Fingerprint />,
+  [Keychain.BIOMETRY_TYPE.FACE]: <Face />,
+  [Keychain.BIOMETRY_TYPE.IRIS]: <Face />,
```

Comment:
Doesn't matter now, but should we have an iris icon?

Reply:
update: Nitya couldn't find an iris icon. let's address this when we plan the rollout for android, she suggested asking Tinka/Bryan for an icon but it wouldn't be the standard android icon